src/main/kotlin/uk/gov/justice/digital/hmpps/authorizationserver/resource/ClientsController.kt

```diff
@@ -0,0 +1,24 @@
+package uk.gov.justice.digital.hmpps.authorizationserver.resource
+
+import org.springframework.http.ResponseEntity
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.stereotype.Controller
+import org.springframework.web.bind.annotation.GetMapping
+import uk.gov.justice.digital.hmpps.authorizationserver.service.ClientSummary
+import uk.gov.justice.digital.hmpps.authorizationserver.service.ClientsService
+
+@Controller
+class ClientsController(
+  private val clientsService: ClientsService,
+) {
+
+  @GetMapping("clients/all")
+  @PreAuthorize("hasRole('ROLE_OAUTH_ADMIN')")
+  fun list(): ResponseEntity<Any> {
```

Comment:
can u pls create separate ticket to add swagger docs

Reply:
This isn't really an API end point - only client will be the UI